tony-history-server/app/controllers/JobsConfigPageController.java

```diff
@@ -0,0 +1,47 @@
+package controllers;
+
+import com.typesafe.config.Config;
+import java.util.ArrayList;
+import java.util.List;
+import javax.inject.Inject;
+import models.JobConfig;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import play.Logger;
+import play.Logger.ALogger;
+import play.mvc.Controller;
+import play.mvc.Result;
+
+import static utils.HdfsUtils.*;
+import static utils.ParserUtils.*;
+import static utils.SecurityUtils.*;
+
+
+public class JobsConfigPageController extends Controller {
```

Comment:
Looks like this page only shows config for a single job. Can we rename it JobConfigPageController (no 's')?